client/pages/config/index.vue

```diff
@@ -293,7 +299,18 @@ export default {
     },
     dateFormats() {
       return this.$store.state.globals.dateFormats
-    }
+    },
+    timeFormats() {
+      return this.$store.state.globals.timeFormats
+    },
+    dateExample() {
+      const date = new Date(2014, 2, 25);
```

Comment:
Any reason to not use the current date for these? Just to not have confusion?

Reply:
Yes, I choose this specific date and time (to be honest, I'm using the same date example from Sonarr) because it can avoid confusion. For example, day two and month two can be represented 02-02. However, this needs to be clarified.
Choosing a late month day helps to avoid this.